tox.ini

```diff
@@ -9,3 +9,6 @@ deps =
 [testenv:flake8]
 commands = flake8 validator
 deps = flake8~=3.8
+
+[flake8]
+max-line-length = 120
```

Comment:
In qontract-reconcile we agreed on `88`. Should we keep this uniform across our repos?

Reply:
i was somehow under the impression we were at 120. i will revert to 88.

Reply:
[f87fece](https://github.com/app-sre/qontract-validator/pull/37/commits/f87fecebc3cf717084bc423466e3ebc4e691e399)